### PR TITLE
Create amir-mokarchi.json

### PR DESCRIPTION
**Pull request by mokarchi**
# #WeAreDonet Creator Submission Guidelines

* [x] Check all points before submitting the PR.

* [x] By submitting this pull request to add my creator card, I verify that I adhere to the #WeAreDotnet creator card submission guidelines.

* [x] Make sure to follow our [Code of Conduct](https://github.com/wearedotnet/dotnet-creator-cards/blob/master/CODE_OF_CONDUCT.md) 

* [x] We do not accept submissions with the sole intention of promoting a product or company.

* [x]  We add you as a creator, not your company. 

* [x]  The moderator team decides whether we accept or reject a submission.

## When Do We Accept Someone's Creator Card Submission? 

* [x] You must have published at least three content pieces (long form content) since 2022 on any of your channels. 
* .NET itself
* .NET libraries and projects
* .NET languages C#, F#, Visual Basic
* Tools needed during development (for .NET)
* CI/CD
* Only in combination with .NET: SPA Frontend technologies like Angular, Vue.js, React etc. 
* Cloud services related to .NET applications (AWS, Azure...)

* [x] There must be three content pieces in English. 

* [x] There needs to be more than tweets. 

* [x] Longer-form article on LinkedIn is good, but a self-hosted blog or one on a well-known platform like Hashnode, medium etc., is better. 

* [ ] Alternatively, you contributed with another form on a platform like YouTube, Twitch, Udemy or Pluralsight. If you have some recorded talks of conferences or anywhere else, that counts too! So in the form of a recorded video or a book. 

## RSS or Atom Feed 

* [ ] The content with feed.type = dotnet must be .NET related:
* .NET itself
* .NET libraries and projects
* .NET languages C#, F#, Visual Basic
* Tools needed during development (for .NET)
* CI/CD
* Only in combination with .NET: SPA Frontend technologies like Angular, Vue.js, React etc. 
* Cloud services related to .NET applications (AWS, Azure...)

* [ ] the content under feed.type = personal may contain off-topic posts.
* Project management
* leadership
* Other

* [ ] Submit an Atom or RSS URL inside your creator card that belongs to your blog 

* [ ] If you do not submit an RSS or Atom URL. We cannot consider your feed for the aggregation feature. 

## Content Guideline 

* [x] If you have a self-hosted blog, please ensure you have a valid HTTPS certificate. 

* [x] We host NO malicious or offensive content on the blog (including photos, swearing, etc.)

* [x] You must add the full URL to your GitHub profile to your card, so we can assign it to the member account that used the same GitHub account for sign-up.

### CODE REVIEW

File: creator-cards/amir-mokarchi.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "sessionize": ""
+  },
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://medium.com/@mokarchi"
+    }
+  ],
+  "feeds": [{
+      "type": "",
```
Comment (DeeJayTC) [inline creator-cards/amir-mokarchi.json:24]: You gave us an empty feed entry, does your blog have an rss feed?

**Review by DeeJayTC — CHANGES_REQUESTED** [review]
Please fix your feed entry

**Comment by mokarchi** [comment]
> Please fix your feed entry

I fixed it

File: creator-cards/amir-mokarchi.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "sessionize": ""
+  },
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://medium.com/@mokarchi"
+    }
+  ],
+  "feeds": [{
+      "type": "Feed Blog",
```
Comment (DeeJayTC) [inline creator-cards/amir-mokarchi.json:24]: Hey @mokarchi this has to be either "dotnet" or "personal"

Dotnet -> only contains .NET related stuff (or Azure etc...)
Personal -> if your feed contains non programming related things or other languages like javascript frameworks etc.

Comment (mokarchi) [reply creator-cards/amir-mokarchi.json]: this feed only contains .NET and I modified with dotNet type

**Pull request merged**